src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/data/fetch/fetchFormDataSagas.ts

```diff
@@ -62,6 +63,14 @@ function* fetchFormDataInitialSaga(): SagaIterator {
         // backward compatibility for https://github.com/Altinn/altinn-studio/issues/6227. Support for nugets < 4.7.0
```

Comment:
no apps in altinn.studio that reference nuget 4.7.0, so we might remove this clause to reduce complexity